### ISSUE TITLE
is it possible to change the loss function in classification?

### ISSUE THREAD

**Issue by fyears** [MinishLab/model2vec#298]
Obviously, in https://github.com/MinishLab/model2vec/blob/e0bb55f38126e0feace5a0f65f51b6dd870bc8d0/model2vec/train/classifier.py#L407

the loss function is cross entropy.

But I would like to use another loss function (focal loss for imbalanced data)

Is it possible?
